staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/subresource.go

```diff
@@ -47,16 +47,7 @@ func NewSubresourceScaler(parentEntries []metav1.ManagedFieldsEntry, parentGroup
 	// The assumption is that scale objects have autoscaling/v1 group version and
 	// corresponding .spec.replicas path
 	h.subresourceGroupVersion = schema.GroupVersion{Group: "autoscaling", Version: "v1"}
-	subresourcePath, err := fieldpath.MakePath("spec", "replicas")
-	if err != nil {
-		return nil, err
-	}
-	h.subresourcePath = subresourcePath
-
-	parentPath, err := fieldpath.MakePath(parentPathPieces...)
-	if err != nil {
-		return nil, err
-	}
+	h.subresourcePath = fieldpath.MakePathOrDie("spec", "replicas")
```

Comment:
That's much simpler. Do we still need this to be part of `h` since the value is always the same?